Source/AssistantV2/Assistant.swift

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2018, 2019.
+ * (C) Copyright IBM Corp. 2018, 2020.
```

Comment:
You should avoid copyright changes if nothing else in the file changed.

Reply:
This is a consequence of the hand edit system. If we `cherry-pick` hand edits over new generations, it will bring that forward in the git history, and then the copyright date generator will apply the current year to the copyright.

Reply:
Okay ... fair enough.  But you should still change the copyright back if nothing has actually changed in 2020.

Maybe do this by undoing the change by the generator, so that the git history is consistent with the copyrights.